### PR TITLE
Fix: #AR-8274 session keys update

### PR DESCRIPTION
**Pull request by shaloo**
Readme and usage guide update for session keys.

### CODE REVIEW

File: usage.md [hunk 1]
```diff
@@ -66,6 +67,86 @@ Returns the Smart Address as per ERC-4337, the SCW address, associated with the
...
+
+```ts
+  import { SCW, StorageType  } from "@arcana/scw";
+
+  let scWallet: SCW;  
+
+  type CreateSessionParam = {
+    contractAddress: string;
+    functionSelector: string;
+    rules: string[];
```
Comment (steel-feel) [inline usage.md]: It a array of `Rule` object (imported from biconomy)

File: usage.md [hunk 2]
```diff
@@ -66,6 +67,86 @@ Returns the Smart Address as per ERC-4337, the SCW address, associated with the
 
 Returns the balance available in the gas tank that can be utilized via the current user's SCW address.
 
+### `createSession()`
+
+Creates session keys using the [`CreateSessionParam`](https://gasless-sdk-ref-guide.netlify.app/types/createsessionparam) policy and requests the user for permission. When the user initiates any app operation that fulfills the session policy, it will be autonomously approved.
```
Comment (steel-feel) [inline usage.md]: Should we mention SCW method name i.e. `createSession` instead of its parameter type ?

**Review by steel-feel — COMMENTED** [review]
Added comments

File: usage.md (hunk 1 above)
Comment (shaloo) [reply usage.md]: fixed.

File: usage.md (hunk 2 above)
Comment (shaloo) [reply usage.md]: Yes indeed my bad.

Comment (shaloo) [reply usage.md]: fixed

File: usage.md [hunk 3]
```diff
@@ -69,7 +69,11 @@ Returns the balance available in the gas tank that can be utilized via the curre
 
 ### `createSession()`
 
-Creates session keys using the [`CreateSessionParam`](https://gasless-sdk-ref-guide.netlify.app/types/createsessionparam) policy and requests the user for permission. When the user initiates any app operation that fulfills the session policy, it will be autonomously approved.
+Creates session keys using the `CreateSession` method and specify the session policy via [`CreateSessionParam`](https://gasless-sdk-ref-guide.netlify.app/types/createsessionparam) parameter.
```
Comment (shaloo) [inline usage.md]:
```suggestion
Create session keys using the `CreateSession` method and specify the session policy via [`CreateSessionParam`](https://gasless-sdk-ref-guide.netlify.app/types/createsessionparam) parameter.

**Review by steel-feel — APPROVED** [review]

**Pull request merged**